src/image/jpeg_io.cpp

```diff
@@ -82,6 +82,10 @@ void decode_jpeg(const char* data, size_t length, char** out_data, size_t& out_l
     jpeg_read_header(&cinfo, TRUE); // Read file header, set default decompression parameters
     jpeg_start_decompress(&cinfo); // Start decompressor
 
+    if (cinfo.out_color_space != JCS_GRAYSCALE && cinfo.out_color_space != JCS_RGB) {
+      log_and_throw(std::string("Image provided is not in supported colorspaces Grayscale or RGB."));
```

Comment:
Unsupported colorspace format. Currently, only RGB and Grayscale are supported.

Reply:
Thank you, this is better phrasing! I'll edit it.